### DISCUSSION TITLE
can't prefore pick_coordinates on other ports

### DISCUSSION THREAD

**Discussion by ShiranGlasser** [dov/giv#69] [Q&A]
Hi, I'm trying to run pick coordinate on ubuntu.
When I'm opening the image at the port 8448 (by not mentioning a port and choosing the default, also by mentioning 8448 port by adding --port 8448), the method pick_coordinate works. but when I choose any other port, the method does'nt work.
It does connect to the wanted port (when I call the method with a port different than the port I opened the giv image with, it causes connection refuse) 
so, calling the method on the image's port- other then 8448, does connect to the port, but unlike opening image in port 8448, it doesn't do anything, there isn't an option on the image to pick coordinates.

Could you please explain what I can do to fix it?
Thank you

**Comment by dov** [comment]
You didn't mention which port you tried to use. Note that on Linux/Unix, a non-root user cannot open and listen to a port below 1024. Try e.g. using port 8449 and verify that it works.

See: https://www.w3.org/Daemon/User/Installation/PrivilegedPorts.html

**Reply by dov** [reply]
Did it help?

**Reply by ShiranGlasser** [reply]
yes! it works with the suggested ports except 8080
Thank you again
